packages/core/parcel-bundler/src/Bundler.js

```diff
@@ -154,6 +154,8 @@ class Bundler extends EventEmitter {
           : isProduction,
       throwErrors:
         typeof options.throwErrors === 'boolean' ? options.throwErrors : true,
+      serveIndex:
```

Comment:
I don't think this should be an option, seems logical to me that it would return the `index.html` file if there is one.

Reply:
Resolved